taipy/core/_scheduler/_scheduler_factory.py

```diff
@@ -17,10 +18,16 @@
 
 class _SchedulerFactory:
     @classmethod
-    def _build_scheduler(cls):
+    def _build_scheduler(cls) -> _AbstractScheduler:
         if Config.job_config._is_default_mode():
-            _Scheduler._recover_jobs()
-            return _Scheduler
-
-        package = f"taipy.{Config.job_config.mode}.scheduler"
-        return _load_fct(package, "Scheduler")
+            if util.find_spec("taipy.enterprise"):
+                # Mode standalone with enterprise version
+                package = "taipy.enterprise.core.scheduler"
+                return _load_fct(package, "Scheduler")()
+            else:
+                # Mode standalone with community version
+                return _Scheduler()
+        else:
+            # Mode airflow with enterprise version
+            package = f"taipy.{Config.job_config.mode}.scheduler"
+            return _load_fct(package, "Scheduler")()
```

Comment:
https://github.com/Avaiga/taipy-core/blob/develop/taipy/core/config/job_config.py

Reply:
not sure I understand what you mean?

Reply:
🤣

Reply:
He meant that we could re-use _MODE_TO_MODULE to get the package name from the mode.
```
_MODE_TO_MODULE: Dict[str, str] = {
    "airflow": "taipy.airflow",
}
```

Reply:
ahhh i see i see, we surely can!